packages/dds/merge-tree/src/client.ts

```diff
@@ -966,9 +966,19 @@ export class Client {
         }
     }
 
-    getContainingSegment<T extends ISegment>(pos: number) {
-        const segWindow = this.mergeTree.getCollabWindow();
-        return this.mergeTree.getContainingSegment<T>(pos, segWindow.currentSeq, segWindow.clientId);
+    getContainingSegment<T extends ISegment>(pos: number, op?: ISequencedDocumentMessage) {
+        let seq;
+        let clientId;
```

Comment:
nit: I prefer explicit typing on declarations like this because that way I don't have to go hunting later in the code to deduce the types.

Reply:
though I guess seq and clientId are common concepts in this codebase.